{"uuid": "e02b1c03-1185-46ee-bcd3-6a91e926cd77", "children": ["6018651a-6e8b-403b-8d2c-11b05677424f", "34b5634f-0a4a-432b-bb49-4920d989e2b0", "bad2599d-725e-42f4-9731-ec787f0f490a"], "befores": [{"name": "setup_teardown", "status": "passed", "start": 1692027689758, "stop": 1692027693012}], "afters": [{"name": "setup_teardown::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=115.0.5790.171)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00007FF7D4DF4A62+57106]\n\t(No symbol) [0x00007FF7D4D6CF52]\n\t(No symbol) [0x00007FF7D4C3E2CB]\n\t(No symbol) [0x00007FF7D4C2A776]\n\t(No symbol) [0x00007FF7D4C2AEAE]\n\t(No symbol) [0x00007FF7D4C2ADE0]\n\t(No symbol) [0x00007FF7D4C1CA8A]\n\t(No symbol) [0x00007FF7D4C1D66D]\n\t(No symbol) [0x00007FF7D4CA06D4]\n\t(No symbol) [0x00007FF7D4C96980]\n\t(No symbol) [0x00007FF7D4C6CE81]\n\t(No symbol) [0x00007FF7D4C6E064]\n\tGetHandleVerifier [0x00007FF7D50A4222+2873042]\n\tGetHandleVerifier [0x00007FF7D50F6590+3209792]\n\tGetHandleVerifier [0x00007FF7D50EF3AF+3180639]\n\tGetHandleVerifier [0x00007FF7D4E85F25+652245]\n\t(No symbol) [0x00007FF7D4D78618]\n\t(No symbol) [0x00007FF7D4D747C4]\n\t(No symbol) [0x00007FF7D4D748BC]\n\t(No symbol) [0x00007FF7D4D64C33]\n\tBaseThreadInitThunk [0x00007FFE45F07614+20]\n\tRtlUserThreadStart [0x00007FFE47D826B1+33]\n\n", "trace": "  File \"G:\\automation\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"G:\\automation\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"G:\\automation\\test_vcm_automation\\Test_base\\test_EnvironmentSetup.py\", line 25, in setup_teardown\n    self.driver.close()\n  File \"G:\\automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 456, in close\n    self.execute(Command.CLOSE)\n  File \"G:\\automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"G:\\automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1692027738060, "stop": 1692027738151}], "start": 1692027689758, "stop": 1692027738151}